src/components/QuestionEdit/index.tsx

```diff
@@ -61,7 +61,10 @@ export default function QuestionEdit({
       return;
     }
 
-    saveQuestion(data);
+    saveQuestion({
+      ...data,
+      points: Math.ceil(data.points),
```

Comment:
when hitting enter points are coming as string, So making them number

Reply:
Why don't we use `parseInt(data.points, 10)`?

Reply:
parseInt will convert `'49.99999999'` into 49. So. the issue will remain same